tests/conftest.py

```diff
@@ -332,17 +328,16 @@ def _get_height(b):
 
 
 @pytest.fixture
-def inputs(user_pk, b):
+def inputs(user_pk, b, alice):
     from bigchaindb.models import Transaction
-
     # create blocks with transactions for `USER` to spend
     for block in range(4):
         transactions = [
             Transaction.create(
-                [b.me],
+                [alice_pubkey(alice)],
                 [([user_pk], 1)],
                 metadata={'msg': random.random()},
-            ).sign([b.me_private]).to_dict()
+            ).sign([alice_privkey(alice)]).to_dict()
```

Comment:
There are more usages of `b.me` and `b.me_private`. Do you want to stop here to avoid the PR growing even bigger?

Reply:
I will address them in a separate PR.